### DISCUSSION TITLE
Welcome to peridigm Discussions!

### DISCUSSION THREAD

**Discussion by johntfoster** [peridigm/peridigm#140] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jritta** [comment]
Hello everyone,

My name is Jonas Ritter and I am a second year PhD student at the Friedrich-Alexander-Universität Erlangen-Nürnberg (FAU) in Germany. I studied mechanical engineering and now work in the field of material science with specification on porous materials.
I want to enhance Peridigm, find and fix bugs, maybe contribute new features and benefit of the communities knowledge.

Cheers,
Jonas

**Comment by akartri** [comment]
Is the building peridigm dependent on the the architecture of the processor? I am using a windows 10 machine (windows subsystem for Linux), AMD Ryzen processor 8 cores and 16 GB ram. After successful configuring of  openmpi 2.0.1, boost 1_71_0, hdf5-1.8.15, netcdf-c-4.7.1, trilinos-12-10-1, I am getting error while building peridigm ([ 58%] Built target PeridigmLib make: *** [all] Error 2). I have posted the detailed issue [here](https://github.com/peridigm/peridigm/issues/147).

**Reply by oldninja** [reply]
This is likely a library path issue. I also recall having problems with openmpi 2 and up versions. Here are the versions for the libraries I installed: 1) openmpi-1.7.5; 2) hdf5-1.8.16; 3) NetCDF-4.4.5; 4) yamlcpp-0.6.3; 5) trilinos-12.12.1; I skip boost since it is no longer required for Peridigm versions 1.5.0 and up.

**Reply by akartri** [reply]
which linux distro did you use?

**Reply by oldninja** [reply]
I have used it on Ubuntu 16.04, 18.04, 20.04, and also on a Red Hat cluster. I don't think it is an issue related to the distros.

**Reply by akartri** [reply]
I see. Thanks. Can you also share the gcc version you are using?

**Reply by oldninja** [reply]
I have used multiples over time. Started with gcc 4.8.5, then 5.3, 7.3, and 9.2. 
I followed the installation process outlined in [this guideline](https://github.com/PeriDoX/PeriDoX/tree/master/Doc/Peridigm_Installation_Guide).

**Reply by akartri** [reply]
I have tried following the guideline. I have tried with the versions suggested by you as well as in the installation guide. I am still encountering the same problem. Please find the detailed post about it [here](https://github.com/peridigm/peridigm/issues/147#issuecomment-808428250). Please advice.

**Comment by djlittl** [comment]
These are link errors that don’t appear to be related to Peridigm per se.

Have you tried troubleshooting your library paths, etc.?

https://www.google.com/search?client=safari&rls=en&q=/usr/bin/ld:+cannot+find+-lgcc&ie=UTF-8&oe=UTF-8

Dave

From: Rohan ***@***.***>
Reply-To: peridigm/peridigm ***@***.***>
Date: Wednesday, March 24, 2021 at 8:02 AM
To: peridigm/peridigm ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [EXTERNAL] Re: [peridigm/peridigm] Welcome to peridigm Discussions! (#140)


Is the building peridigm dependent on the the architecture of the processor? I am using a windows 10 machine (windows subsystem for Linux), AMD Ryzen processor 8 cores and 16 GB ram. After successful configuring of openmpi 2.0.1, boost 1_71_0, hdf5-1.8.15, netcdf-c-4.7.1, trilinos-12-10-1, I am getting error while building peridigm ([ 58%] Built target PeridigmLib make: *** [all] Error 2). I have posted the detailed issue here<https://github.com/peridigm/peridigm/issues/147>.

**Comment by ppandit95** [comment]
Hello Everyone,

I am Pushkar Pandit, a post graduate mechanical engineer  currently working on a project involving peridynamics in Indian Institute of Technology,Hyderabad,India . I look forward to work with peridigm in order to simulate damage in  functionally graded materials as well as I am interested in different prospects of this theory in multiscale modelling as well as its utility in computational materials science.

Cheers
Pushkar
